receiver/rabbitmqreceiver/documentation.md

```diff
@@ -29,6 +29,3 @@ metrics:
 | Name | Description |
 | ---- | ----------- |
 | message.state | The state of messages in a queue. |
-| rabbitmq.node.name | The name of the RabbitMQ node. |
-| rabbitmq.queue.name | The name of the RabbitMQ queue. |
-| rabbitmq.vhost.name | The name of the RabbitMQ vHost. |
```

Comment:
It looks like the documentation generator needs updating here.

Reply:
same here